css/templates/travel/elements/_input-range.css

```diff
@@ -0,0 +1,75 @@
+
+/*
+ * input-range
+ *
+ * Create a stylized input[type=range] element.
+ */
+.travel-input-range {
+  display: block;
+  background-color: transparent;
+  color: inherit;
+  background-color: transparent;
+  -webkit-appearance: none;
```

Comment:
make the properties with - the last set of properties

Reply:
done

Reply:
the property is still not the last property?